app/assets/javascripts/app/views/publisher/uploader_view.js

```diff
@@ -86,10 +85,10 @@ app.views.PublisherUploader = Backbone.View.extend({
     if (response.success){
       this.info.text(Diaspora.I18n.t("photo_uploader.completed", {file: fileName})).fadeTo(2000, 0);
 
-      var id  = response.data.photo.id,
+      var photoId = response.data.photo.id,
           url = response.data.photo.unprocessed_image.url;
 
-      this._addFinishedPhoto(id, url);
+      this._addFinishedPhoto(_id, photoId, url);
```

Comment:
The `_` at the beginning indicates that the parameter is unused in the function. Since you use it here now, you should remove the `_`.